automatic/vs-extensions/sonarlint-vs2015/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,14 @@
+﻿$ErrorActionPreference = 'Stop';
+
+$toolsPath = Split-Path $MyInvocation.MyCommand.Definition
+$filePath = "$toolsPath\SonarLint.VSIX-2.9.0.355-2015.vsix"
+
+$vsixUrl =  "file://" + $filePath.Replace("\", "/")
```

Comment:
If I remember correctly, I believe @ferventcoder said that although this works, it isn't really supported.